tiledb/sm/c_api/tiledb.cc

```diff
@@ -2830,10 +2830,7 @@ int32_t tiledb_array_get_non_empty_domain_from_name(
     return TILEDB_ERR;
 
   bool is_empty_b;
-
-  throw_if_not_ok(ctx->storage_manager()->array_get_non_empty_domain_from_name(
-      array->array_.get(), name, domain, &is_empty_b));
-
+  array->array_->non_empty_domain_from_name(name, domain, &is_empty_b);
```

Comment:
We need validate arguments before this call.
Same for the two changed calls below.